index.js

```diff
@@ -18,27 +29,30 @@ const manageRecipes = async () => {
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
 
+    // Iteration 2 - Create a recipe
+    await Recipe.create(newRecipe);
+
+    // Iteration 3 - Insert multiple recipes
+    let allRecipes = await Recipe.insertMany(data);
+    console.log(allRecipes);
```

Comment:
If you wanted to `console.log` only the titles of the recipes, you needed to loop over your `allRecipes` variable that is storing an array with your recipes objects.
Something like this:
```
// console log to see the titles of the inserted recipes
    for (let i = 0; i < allRecipes.length; i++) {
      console.log(allRecipes[i].title);
    }
```